FHEM/lib/SD_Protocols.pm

```diff
@@ -1743,6 +1743,41 @@ sub _xFSK_methods_behind_here {
   # only for functionslist - no function!
 }
 
+=item ConvBresser_5in1()
+
+This function checks number/count of set bits within bytes 14-25 and inverted data of 13 byte further.
+Delete inverted data (nibble 1-27)and reduce message length (nibble 53).
+
+Input:  $hexData
+Output: $hexData
+        scalar converted message on success 
+        or array (1,"Error message")
+
+=cut
+
+sub ConvBresser_5in1 {
+  my $self    = shift // carp 'Not called within an object';
+  my $hexData = shift // croak 'Error: called without $hexdata as input';
+  my $d2;
+  my $bit;
+  my $bitsumRef;
+  my $bitadd = 0;
+  for (my $i = 0; $i < 13; $i++) {
```

Comment:
Fehlt hier nicht noch eine Prüfung ob $hexData auch die nötige Länge hat? 
Meiner groben Einschätzung nach müsste die Länge mind. 26 Zeichen sein.

Reply:
Müssen wir das wirklich zweimal prüfen?
In SD_ProtocolData.pm gibt es bereits diese Mindestlänge:

`        length_min      => '52',
`

Reply:
Ich hab gerade die SUB mit einem ? aufgerufen als Eingangsparameter.

Die Fehlermeldung brauche ich nicht posten oder?

Reply:
Nein, brauchst du nicht.
Willst du wirklich eine zweite Prüfung auf length_min, nur weil jemand mutwillig zu kurze Nachrichten direkt an die SUB übergibt?

Reply:
Würd ich machen ja, wer weiss was wir demnächst mal in dem Protokoll warum auch immer anpassen.